app/build.gradle

```diff
@@ -30,6 +36,7 @@ dependencies {
     implementation project(':domain')
     implementation project(':data')
     implementation "androidx.cardview:cardview:1.0.0"
```

Comment:
add these dependencies to the generalDependencies please

Reply:
done